dm/config/task.go

```diff
@@ -539,6 +559,42 @@ func (c *TaskConfig) adjust() error {
 		return terror.ErrConfigDuplicateCfgItem.Generate(strings.Join(duplicateErrorStrings, "\n"))
 	}
 
+	for route := range c.Routes {
+		if gobalConfigReferCount[prefixs[0]+route] == 0 {
+			log.L().Error("route in gobal configuration isn't used", zap.String("route", route))
+			return terror.ErrConfigGobalConfigUnused.Generate()
```

Comment:
we should return the unused configs in error message, grep the names in log is a hard work.

could add the names in a slice and format error message with all of them.